packages/create/src/tracing/logger.ts

```diff
@@ -0,0 +1,64 @@
+import chalk from 'chalk';
+import { ConsoleTransport, LogLevel, ToolsLogger } from '@sap-ux/logger';
+
+const levelColor: { [level: string]: string } = {
+    warn: 'yellow',
+    error: 'red',
+    verbose: 'blue',
+    debug: 'gray'
+};
+
+let logger: ToolsLogger;
+
+/**
+ * Return the logger.
+ *
+ * @returns - Logger
+ */
+export function getLogger(): ToolsLogger {
+    if (!logger) {
+        logger = new ToolsLogger({ transports: [new ConsoleTransport()], logPrefix: '' });
+        setCustomFormatter(logger);
+    }
+    return logger;
+}
+
+/**
+ * Set custom formatting to the logger. Needs better solution, perhaps proxy transport in @sap-ux/logger.
+ *
+ * @param logger - instance of the logger
+ */
+function setCustomFormatter(logger: ToolsLogger): void {
+    const transports = (logger as any)?._logger?.transports;
```

Comment:
That look like something you shouldn't be doing...
Wouldn't it be cleaner if you create a subclass of `ToolsLogger` here and then add/overwrite the things you need? This very same class could also offer something like an `getInstance` to achieve the singleton pattern that implemented.

Reply:
Yes, I'm well aware of this ;). I discussed with @ullasholla, he plans to come up with a concept to provide custom formatters to the logger. I will change to the new approach as soon as it is available.

Reply:
Will be tracked in issue - https://github.com/SAP/open-ux-tools/issues/831